configure.ac

```diff
@@ -67,6 +67,10 @@ m4_if(
 
 AM_CONDITIONAL([HAVE_VERSION_SCRIPT], [test "x$have_version_script" = "xyes"])
 
+AC_ARG_WITH([tpm1],
+    AS_HELP_STRING([--with-tpm1], [build libtpms with TPM1 support]), [], [with_tpm1=yes])
+AM_CONDITIONAL([WITH_TPM1], [test "x$with_tpm1" != "xno"])
+AS_IF([test "x$with_tpm1" != "xno"], [AC_DEFINE([WITH_TPM1], [1], [With TPM 1.2 support])])
```

Comment:
I wonder whether there should be a variable like this in libtpms.pc as well so that swtpm knows it cannot run the TPM 1.2 tests.

Reply:
Sure, that could help to avoid run time skip.

Reply:
done